scripts/test-all-factory

```diff
@@ -25,6 +25,14 @@ testImplementation http-interop/http-factory-guzzle "^1.0" || exit 1
 testImplementation http-interop/http-factory-slim "^1.0" || exit 1
 testImplementation zendframework/zend-diactoros "^2.0" || exit 1
 
+if [[ "$COMPOSER_FLAGS" == *"--prefer-lowest"* ]]
+then
+    echo "Sunrise package missed due to unsupported composer \"--prefer-lowest\" mode"
+    echo
+else
+    testImplementation sunrise/http-factory "^1.0" || exit 1
```

Comment:
Use the lowest version that contains the factor implementation so that this workaround is not necessary.

Reply:
The repository [http-factory](https://github.com/sunrise-php/http-factory) does not contain factories, it pulls other repositories:
[http-message](https://github.com/sunrise-php/http-message), [http-server-request](https://github.com/sunrise-php/http-server-request), [stream](https://github.com/sunrise-php/stream), [uri](https://github.com/sunrise-php/uri)

It's problem... What better way to do?

Reply:
Correct the minimum version requirement in `http-factory` to point to a `stream` implementation that contains the factory.

Reply:
Good solution, thank you very much, all okay?